### DISCUSSION TITLE
[Support]: System metrics > Storage shows a blank page

### DISCUSSION THREAD

**Discussion by sylv3rblade** [blakeblackshear/frigate#11305] [General Support]
### Describe the problem you are having

Bug on v14 

When clicking Storage, you only get a blank page with the ff errors:

![Screenshot 2024-05-09 at 9 56 16 AM](https://github.com/blakeblackshear/frigate/assets/467007/ee1491d2-27f4-44ea-98c1-b497d7eca7d3)

the /storage endpoint seems to be working fine
![Screenshot 2024-05-09 at 10 00 04 AM](https://github.com/blakeblackshear/frigate/assets/467007/3a0e318d-e936-4444-81fd-00e8734da34e)

### Version

0.14.0-ff2948a7

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Tapo 325WB

**Comment by hawkeye217** [comment]
Fixed in https://github.com/blakeblackshear/frigate/pull/11312
